src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/SmallFullServiceImplTests.java

```diff
@@ -0,0 +1,146 @@
+package uk.gov.companieshouse.web.accounts.service.smallfull.impl;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.smallfull.SmallFullResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.request.SmallFullCreate;
+import uk.gov.companieshouse.api.handler.smallfull.request.SmallFullGet;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullApi;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SmallFullServiceImplTests {
+
+    @Mock
+    private ApiClient apiClient;
+
+    @Mock
+    private ApiClientService apiClientService;
+
+    @Mock
+    private SmallFullResourceHandler smallFullResourceHandler;
+
+    @Mock
+    private SmallFullCreate smallFullCreate;
+
+    @Mock
+    private SmallFullGet smallFullGet;
+
+    @InjectMocks
+    private SmallFullService smallFullService = new SmallFullServiceImpl();
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    @BeforeEach
+    private void init() {
```

Comment:
`init` isn't a widely recognised method for setting up the unit tests. `setUp` is more known.